src/Microsoft.Azure.SignalR/ServiceOptions.cs

```diff
@@ -120,5 +120,12 @@ public int ConnectionCount
         /// Gets or sets a function which accepts <see cref="HttpContext"/> and returns a bitmask combining one or more <see cref="HttpTransportType"/> values that specify what transports the service should use to receive HTTP requests.
         /// </summary>
         public Func<HttpContext, HttpTransportType> TransportTypeDetector { get; set; } = null;
+
+        /// <summary>
+        /// Allow clients enable stateful reconnects for connecting to service.
+        /// By default is disabled.
+        /// It can also be configurate <see cref="HttpConnectionDispatcherOptions"/> by hub in net 8.
```

Comment:
also describe how to use it, for example, customer needs a net8 client?

Reply:
Add a feature doc link?

Reply:
I will update it later.